### ISSUE TITLE
Investigate too much contrast

### ISSUE THREAD

**Issue by Camulos** [digdir/designsystemet#4007]
As with how the dark-mode colours are generated using not fully black as the darkest, similarly it may be beneficial to not use fulle white as the lightest colour in light-mode.

It is also possible that fulle black and white should always be avoided.

https://www.w3.org/TR/wcag-3.0/#maximum-text-contrast
https://accessibility.huit.harvard.edu/use-sufficient-color-contrast
https://okse.no/artikler/wcag3
https://www.a11y-collective.com/blog/common-mistakes-with-using-colour-in-accessibility/

Also in this interview we are given some examples on how light is a "bigger" part of that visuals we see than colour:
https://www.youtube.com/watch?v=E-CZial5VkU

Suggested experiments and further investigation:
- Always stay at least 2 % off of either extreme (minimum 2 % light and maximum 98 % light) - how would this change the theme-builder?
- Investigate (further desk-study) what is considered to be too much contrast, is there a value in either /both wcag 2 and APCA contrast calculations that can be used as a baseline?
